asv.conf.json

```diff
@@ -0,0 +1,162 @@
+{
+    // The version of the config file format.  Do not change, unless
```

Comment:
This actually works? This must be some custom JSON parsing by ASV because comments aren't part of the JSON standard.

Reply:
Yeah, that's the template file they provide, but it works...